server/routes/reviewPlan/reviewNoteController.ts

```diff
@@ -19,7 +19,11 @@ export default class ReviewNoteController {
 
     getPrisonerContext(req.session, prisonNumber).reviewNoteForm = undefined
 
-    const view = new ReviewNoteView(prisonerSummary, reviewNoteForm)
+    const backlinkUrl = req.session.pageFlowHistory
```

Comment:
Set backlink url based on PageFlowHistory, which will include the check your answers page if arriving from there.

Reply:
Changed this to a boolean to make things simpler.